docker-hub/image-management.md

```diff
@@ -24,11 +24,14 @@ With the new Advanced Image Management dashboard, all of this is now simpler and
 
 To access the  Advanced Image Management dashboard:
 
-1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard. It lists images in your repository and allows you to sort and filter images by tags, activity status, and the last activity date.
+1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
+2. Click **General** and click **Go to Advanced Image Management** from the main menu, and select a repository from the list.
```

Comment:
**suggestion:** we've removed a critical step here which was this one:

> 2. Click **Repositories** from the main menu and select a repository from the list

Without it, the current step list doesn't make much sense 🤔 This is still needed before we go to the **General** tab, since it only exists within a repository

Reply:
Updated